$NetBSD: patch-ax,v 1.4 2014/08/25 08:23:22 wiz Exp $

Fix unportablet test(1) construct.
Will be included in next Advanced release.

--- converter/ppm/hpcdtoppm/pcdovtoppm.orig	2004-10-23 00:12:12.000000000 +0100
+++ converter/ppm/hpcdtoppm/pcdovtoppm	2006-11-19 15:46:03.000000000 +0000
@@ -195,7 +195,7 @@
     rowfiles="$rowfiles $rowfile"
 fi
 
-if [ ${#rowfiles[*]} == 1 ]; then
+if [ ${#rowfiles[*]} = 1 ]; then
     cat $rowfiles
 else
     if [ "$colors" = "n" ] ; then
